app/assets/javascripts/samples/sample_datatable.js

```diff
@@ -74,6 +81,7 @@ table = $("#samples").DataTable({
     fnDrawCallback: function(settings, json) {
         animateSpinner(this, false);
         changeToViewMode();
+        sampleInfoListener();
```

Comment:
I imagine that this callback is called everytime the table is redrawn, e.g. when user sorts the table, navigates to a different navigation page, etc. (basically, when the rows are reloaded via AJAX from server).

I believe there's (albeit very improbable) potential for a JavaScript memory leak here, since `$(".sample_info").on("click", <handler>);` event handlers are kept in memory for elements that have been removed. 

So I don't know, I'd check if maybe those handlers can be removed (`$(".sample_info").off("click");`) before re-drawing the table (maybe the `fnPreDrawCallback` can be used, needs to be checked).